src/react-queries/index.ts

```diff
@@ -18,6 +31,102 @@ import { thorchainLp } from 'pages/ThorChainLP/queries/queries'
 export const thorchainBlockTimeSeconds = '6.1'
 const thorchainBlockTimeMs = bn(thorchainBlockTimeSeconds).times(1000).toNumber()
 
+const common = createQueryKeys('common', {
+  allowanceCryptoBaseUnit: (
+    assetId: AssetId | undefined,
+    spender: string | undefined,
+    from: string | undefined,
+  ) => ({
+    queryKey: ['allowanceCryptoBaseUnit', assetId, spender, from],
+    queryFn: async () => {
+      if (!assetId) throw new Error('assetId is required')
+      if (!spender) throw new Error('spender is required')
+      if (!from) throw new Error('from address is required')
```

Comment:
Paranoia and type exhaustiveness - these will never get hit because of  the `enabled` flag